docs/how-to-guides/openstack-swift.md

```diff
@@ -0,0 +1,34 @@
+# Content-cache with Openstack/Swift storage
+
+Sometimes it is desirable to cache swift storage objects inside the charm for faster processing and reducing the number of requests to the [swift](https://docs.openstack.org/swift/latest/) server.
```

Comment:
I think we should mention that another reason is you might want to have different hostname than the one of the Swift service. That's probably the main reason people would want to use this feature, I think.